src/SmartTransactionsController.ts

```diff
@@ -718,38 +775,35 @@ export default class SmartTransactionsController extends StaticIntervalPollingCo
       );
     }
     transactions.push(unsignedTradeTransactionWithNonce);
-    const data = await this.fetch(getAPIRequestURL(APIType.GET_FEES, chainId), {
-      method: 'POST',
-      body: JSON.stringify({
-        txs: transactions,
-      }),
-    });
-    let approvalTxFees;
-    let tradeTxFees;
+    const data = await this.#fetch(
+      getAPIRequestURL(APIType.GET_FEES, chainId),
+      {
+        method: 'POST',
+        body: JSON.stringify({
+          txs: transactions,
+        }),
+      },
+    );
+    let approvalTxFees: IndividualTxFees | null = null;
```

Comment:
nit:
```suggestion
    let approvalTxFees: IndividualTxFees | null;
```

Reply:
They should actually be null and not undefined as undefined is not serialisable on state